data/mods/Dark-Skies-Above/overrides/items/carnivore.json

```diff
@@ -51,7 +51,7 @@
     "copy-from": "mutant_meat",
     "type": "COMESTIBLE",
     "name": { "str": "chunk of unusual meat", "str_pl": "chunks of unusual meat" },
-    "description": "Meat from the aliens is typically quite foul, full of toxins and substances not meant to be digested by humans, however this piece almost looks edible - if not for a few sections that still have strange hues and disgusting, spongey texture.  Still, with a bit of preperation, it might even be somewhat palatable."
+    "description": "Meat from the aliens is typically quite foul, full of toxins and substances not meant to be digested by humans, however this piece almost looks edible - if not for a few sections that still have strange hues and disgusting, spongy texture.  Still, with a bit of preparation, it might even be somewhat palatable."
```

Comment:
```suggestion
    "description": "Meat from the aliens is typically quite foul, full of toxins and substances not meant to be digested by humans; however, this piece almost looks edible - if not for a few sections that still have strange hues and disgusting, spongy texture.  Still, with a bit of preparation, it might even be somewhat palatable."
```